posts/Stef/2019-11-18-hibernate-orm-with-panache-in-quarkus.adoc

```diff
@@ -0,0 +1,462 @@
+= Hibernate ORM with Panache in Quarkus
+Stephane Epardaud
+:awestruct-tags: [ "Hibernate ORM", "Quarkus" ]
+:awestruct-layout: blog-post
+---
+
+We have talked about this link:https://quarkus.io/[Quarkus] “Panache” thing in  
+link:/2019/11/12/hibernate-search-quarkus/[recently in this blog],
+but we should probably get into a little more detail about what it is, and why it matters.
+
+First thing, to clarify, we're talking about “Hibernate ORM with Panache”, which roughly means we're talking about a 
+https://en.wikipedia.org/wiki/Panache[flamboyant and reckless] flavour of Hibernate ORM.
+
+This means two essential things:
+
+- It's the full Hibernate ORM, not anything less
+- But it look different to what you're used to
```

Comment:
- You can fallback at any time and even mix vanilla JPA and Panache

Reply:
Good point